module4/projects/capstone/expectations.md

```diff
@@ -0,0 +1,180 @@
+
+## Working Like a Developer
+After the onboarding and setup period, you will be assigned your first feature. Some features will be small and others will be large enough that you will need to break them into multiple issues. You should ask clarifying questions with your project manager as needed. There will be daily opportunities to do this synchronously in the daily check in, otherwise you can use Slack to communicate with your project manager or ask to schedule a call. Your project manager will be the final decision maker on the scope of the feature, but you are encouraged to ask questions and provide input.
+
+As the developer, you are responsible for:
+* Planning the feature (see below)
+* Writing the code to implement the feature
+* Testing the feature to verify it works as expected
+* Writing or updating the documentation for the feature
+* Ensuring your code does not break existing functionality or tests
+  * If existing tests break or are inadequate, update them. Be sure to document those changes and why there were needed in the PR.
+* Ensuring CI passes before requesting a review
+* Verifying your deployed changes are functioning as expected in production
+
+### Planning the Feature Looks Like:
+* Clarifying requirements of user stories with the project manager
+* Breaking down the feature into smaller issues if needed AND adding those issues to the project board
+  * Remember, PRs should be focused on a single feature or bug fix. If you are working on a large feature, break it into smaller pieces and submit them as separate PRs. Each PR should have a single purpose and address a single issue.
+* Choosing the appropriate tools, patterns, and technologies to implement the feature
+  * This might include discussing tradeoffs with the project manager or other developers
+
+<section class="dropdown">
+### Breaking Down a Feature Example
+
+If the assigned feature is **Users can sign up for a new account**
+
+You might break it down into the following issues after clarifying requirements with the project manager:
+* Create a back end endpoint for sign up
+* Create a front end component for sign up
+* Create a new route for sign up
+* Add validations to the form
+* Add backend validations to ensure the email is not already in use
+
+Each of these issues should be added to the project board. The issue details should include a description of the feature, any acceptance criteria, and any additional information that will help you complete the feature such as:
+* Links to documentation or code
+* Dependencies (ex: gems or packages)
+* Potential blockers (ex: decisions or issues that need to be completed before this one can be started)
+
+They should be given a status of **Backlog** and the Project Manager will move them to **To Do** when they have been scheduled to be worked on.
+
+**For the example "Create a back end endpoint for sign up":**
+* Description: Create a back end endpoint for sign up
```

Comment:
If students are working on separate applications during these projects, I think separating BE from FE stories makes sense. Just wanted to flag here that we have introduced the idea of "vertically sliced" user stories that are fullstack in the Mod 3 consultancy projects. But TBD if students use them or just break them up FE/BE